src/main/resources/edu/cornell/kfs/module/purap/document/datadictionary/CuPaymentRequestDocument.xml

```diff
@@ -19,56 +17,5 @@
   <bean id="PaymentRequestDocument-bankCode" parent="PaymentRequestDocument-bankCode-parentBean">
       <property name="required" value="false"/>
   </bean>
-  
-  <!-- workflow attributes for routing -->

```

Comment:
We should add paymentMerthodCode back to the search fields. Base code did not add the payment method to the search fields but we used to have it in our customization and it makes PREQs searches easier. I think I removed this when I was cleaning up the code but I now realise we still need it. Thank you.

Reply:
paymentMethodCode search field put back in place. Also fixed formatting in the file.